DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DeletePresetServlet.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
```

Comment:
Can you add a test for the doPost function?

Reply:
Yup, will add that after our sync.